### PR TITLE
#1953: sequencer: completely remove component

### LINKED ISSUE #1953: Remove the dead sequencer code
**What Needs to be Done?**

- [ ] First do this
- [ ] Then do that.
- [ ] Also this other thing.

**Is your feature request related to a problem? Please describe.**

**Describe potential solution outcome**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by lifflander**
Fixes #1953

### CODE REVIEW

File: src/vt/context/runnable_context/set_context.cc [hunk 1]
```diff
@@ -41,6 +41,7 @@
 //@HEADER
 */
 
+#include "vt/context/context.h"
```
Comment (PhilMiller) [inline src/vt/context/runnable_context/set_context.cc:44]: What's this doing here?

**Review by PhilMiller — APPROVED** [review]

File: src/vt/context/runnable_context/set_context.cc (hunk 1 above)
Comment (lifflander) [reply src/vt/context/runnable_context/set_context.cc:44]: When I removed the component some transitive dependency was causing this code to no longer compile.

Comment (lifflander) [reply src/vt/context/runnable_context/set_context.cc:44]: And it does depend on the context so that include should be there.

**Review by nlslatt — CHANGES_REQUESTED** [review]
Some files were missed. At the very least, in `sketches/`.

**Comment by nlslatt** [comment]
This merge appears to have broken the Gitlab CI builds. It looks like it might be a transitive includes problem.

**Pull request merged**